gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversalTest.java

```diff
@@ -71,4 +80,89 @@ public void graphTraversalShouldHaveMethodsOfAnonymousGraphTraversal() {
             }
         }
     }
+
+    @Test
+    public void graphTraversalMethodsShouldGenerateCorrespondingBytecode() throws Exception {
+        final Random random = new Random();
```

Comment:
Who ever does `-DassertNonDeterministic`? If we use that then its basically a non-test as I have never used/seen that flag. The point seems that tests that ARE random should be used as much as possible (`mvn clean install`-style) so you can scan the parameter space more and more and more.